ampd/src/handlers/config.rs

```diff
@@ -202,6 +206,22 @@ where
     }
 }
 
+fn validate_stellar_verifier_set_verifier_config<'de, D>(configs: &[Config]) -> Result<(), D::Error>
+where
+    D: Deserializer<'de>,
+{
+    match configs
+        .iter()
+        .filter(|config| matches!(config, Config::StellarMsgVerifier { .. }))
+        .count()
+    {
+        count if count > 1 => Err(de::Error::custom(
+            "only one Stellar verifier set verifier config is allowed",
+        )),
+        _ => Ok(()),
+    }
+}
```

Comment:
can we create a macro for this, so it's `ensure_unique_config!("Stellar verifier set", Config::StellarMsgVerifier)` and can be reused for all other checks as well, or do these validations differ from config to config?

Reply:
sg, will work on this